RefactorTest/RenameCpp20.cpp

```diff
@@ -456,6 +456,39 @@ void f10()
     }
 }
 
+// using for scoped enums
+void f11()
+{
+    //#TEST#: R1521 Rename E
+    enum class E : uint8_t {
+        //#TEST#: R1522 Rename One
+        One,
+        //#TEST#: R1523 Rename Two
+        Two,
+        Three
+    };
+
+    struct F11 {
+        using enum E;
+    }f11Ins;
+
+    const auto& get_enumerator = [&]() -> auto {
```

Comment:
Odd that you are taking a reference to a temporary.  Not undefined behavior, but just odd.